WrenchFu/gui.lua

```diff
@@ -11,52 +11,65 @@ Tweakable:
 ]]
 
 if not WrenchFu then WrenchFu = {} end
-WrenchFu.max_distance = 6
 
 
-function WrenchFu.open_gui_for(player_index, entity_name, position)
-    local reg = WrenchFu.handler_for(entity_name)
-
-    global.open_guis[player_index] = {
-        ["entity_name"] = entity_name,
-        ["position"] = position,
+function WrenchFu.open_gui_for(player_index, entity)
```

Comment:
Taking an actual entity as a parameter here irrevocably breaks any semblance of backward compatibility, but you're not actually doing anything with it -- you're still only using the entity name and position. Why change?